tests/test_wigner_samples.py

```diff
@@ -59,7 +60,7 @@ def test_elmn2ind(s2fft_to_so3_sampling, L: int, N: int, sampling: str):
 
 @pytest.mark.parametrize("L", L_to_test)
 @pytest.mark.parametrize("N", N_to_test)
-def test_so3_samples(flmn_generator, L: int, N: int):
+def test_so3_samples_2(flmn_generator, L: int, N: int):
```

Comment:
@CosmoMatt This test function has the same name as another function defined earlier in file and so was shadowing this - I have just added a `_2` suffix for now but if only one of these two functions is required we should remove one, or give more useful separate names.

Reply:
good catch!